data/products/sle-micro/5.3/preferences.yaml

```diff
@@ -1,6 +1,7 @@
 preferences:
   type:
     _attributes:
+      image: oem
```

Comment:
Just for the record this is an overwrite of ```image: vmx``` which is present in the defaults.yml. It makes sense here and I'm fine with it. For the "real" change to schema version 7.5 we should get rid of any ```image: vmx``` specification in the recipes though. The same applies to the schemaversion setting.

Reply:
We have a lot of activity going on as such I think it is good to keep the ripples in the image descriptions to a minimum at this time and only apply the change where absolutely necessary.